spec/classes/config/global/storage_converter_spec.rb

```diff
@@ -0,0 +1,42 @@
+# rubocop:disable RSpec/MultipleExpectations
+
+require 'spec_helper'
+
+describe 'rundeck' do
+  context 'supported operating systems' do
+    %w[Debian RedHat].each do |osfamily|
```

Comment:
could you instead use rpsec-puppet-facts? That allows you to easily iterate on all operating systems in metadata.json. An example is https://github.com/voxpupuli/puppet-rundeck/blob/master/spec/classes/config_spec.rb#L6